spec/game_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "game"
+
+RSpec.describe Game do
+  it "can choose a random move" do
+    expect(["rock", "paper", "scissors"]).to include subject.random_move
+  end
+
+  describe "#result" do
+    it "is draw, when same signs" do
+      expect(subject.result("rock", "rock")).to eq "rock vs. rock: it's a draw!"
```

Comment:
According to Makers' review rubric, it's bad practice to have multiple expects in the same "it" block because they should really be separate tests.  I think it makes sense to do it how you have, and it saves a lot of duplicated code, but I just thought I should say.